src/react-components/room/useMicrophone.js

```diff
@@ -1,17 +1,17 @@
-import { useState, useEffect, useRef } from "react";
+import { useState, useEffect, useRef, useCallback } from "react";
 import MovingAverage from "moving-average";
 
-const MOVING_AVG_TIMESPAN = 100;
 const UPDATE_RATE = 50;
 
-export function useMicrophoneVolume(scene) {
+export function useMicrophone(scene, updateRate = UPDATE_RATE) {
```

Comment:
Should this line below use `updateRate` (instead of `UPDATE_RATE`)?
https://github.com/mozilla/hubs/blob/7159d67ccc84d6dfccf7bf3b1590275f04092367/src/react-components/room/useMicrophone.js#L29

Seems like you can remove `UPDATE_RATE` and write `updateRate = 50`.